hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RegionServerAccounting.java

```diff
@@ -123,6 +127,13 @@ public long getGlobalMemStoreOffHeapSize() {
     return this.globalMemStoreOffHeapSize.sum();
   }
 
+  /**
+   * @return the retained metrics of region's read and write requests count
+   */
+  public ConcurrentMap<String, Pair<Long, Long>> getRetainedRegionRWRequestsCnt() {
```

Comment:
Has to be public? Can't be package private?

Reply:
It could be protected, will fix.